bool on
float32 duration # in seconds